File: src/main/java/baboon/industry/block/storage/entity/TileEntityTransformerMVtoLV.java
package baboon.industry.block.storage.entity;

import baboon.industry.IndustryConfig;

public class TileEntityTransformerMVtoLV extends TileEntityTransformerBase {

    public TileEntityTransformerMVtoLV() {
        setCapacity(IndustryConfig.cfg.getInt("Energy Values.lvIO"));
        setTransfer(IndustryConfig.cfg.getInt("Energy Values.lvIO"));
        setMaxReceive(IndustryConfig.cfg.getInt("Energy Values.mvIO"));
    }
}

File: src/main/kotlin/turniplabs/industry/blocks/entities/transformer/TileEntityTransformerMVtoLV.kt
package turniplabs.industry.blocks.entities.transformer

import sunsetsatellite.sunsetutils.util.Connection
import sunsetsatellite.sunsetutils.util.Direction
import turniplabs.industry.IndustryConfig
import turniplabs.industry.blocks.entities.TileEntityEnergyConductorDamageable

class TileEntityTransformerMVtoLV : TileEntityEnergyConductorDamageable() {

    init {
        this.setCapacity(2048)
        this.setTransfer(IndustryConfig.cfg.getInt("Energy Values.lowVoltage"))
        this.setMaxReceive(IndustryConfig.cfg.getInt("Energy Values.mediumVoltage"))

        setConnection(Direction.Y_POS, Connection.INPUT)
        setConnection(Direction.Y_NEG, Connection.OUTPUT)
    }
}